packages/vue/src/components/organisms/SfHeader/SfHeader.vue

```diff
@@ -190,111 +223,41 @@ export default {
       default: "",
     },
     /**
-     * Header wishlist items quantity
-     */
-    wishlistItemsQty: {
-      type: [String, Number],
-      default: "0",
-    },
-    /**
-     * Header cart items quantity
+     * Open sidebar on mobile view
      */
-    cartItemsQty: {
-      type: [String, Number],
-      default: "0",
-    },
-    /**
-     * Header sticky to top
-     */
-    isSticky: {
-      type: Boolean,
-      default: false,
-    },
-    /**
-     * Is nav slot visible on mobile view
-     */
-    isNavVisible: {
-      type: Boolean,
-      default: false,
+    openSidebar: {
+      type: [Boolean, String],
+      default: true,
     },
   },
   data() {
     return {
       icons: [],
-      hidden: false,
       sticky: false,
       scrollDirection: null,
       lastScrollPosition: 0,
       animationStart: null,
       animationLong: null,
-      animationDuration: 300,
+      animationDuration: 3000,
```

Comment:
1. Are you using this reactive property?
2. If this won't change it can be a const i.e. `ANIMATION_DURATION = 3000`

Reply:
no it should be removed. Thanks,